packages/cli/src/services/project.service.ts

```diff
@@ -71,6 +72,23 @@ export class ProjectService {
 		return project;
 	}
 
+	async updateProject(name: string, projectId: string): Promise<Project> {
+		const result = await this.projectRepository.update(
+			{
+				id: projectId,
+				type: 'team',
```

Comment:
We're ignoring `public` projects in our code at the moment

https://github.com/n8n-io/n8n/blob/4ed2d9e83d855200c173cd902a97859b517bf9fc/packages/cli/src/databases/entities/Project.ts#L7

Maybe we should remove that from the type for now.

Reply:
Good point although I don't think it's part of the extent of this PR. We should discuss this with @valya